parachain/pallets/ethereum-light-client/Cargo.toml

```diff
@@ -15,22 +15,23 @@ codec = { version = "2.0.0", package = "parity-scale-codec", default-features =
 hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
 hex-literal = { version = "0.3.1", optional = true }
 rlp = { version = "0.5", default-features = false }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
```

Comment:
Can we order this import so that it comes right after `codec`? Serdec, codec, and scale-info seem to be needed in most pallets so its good to group them together.